dask/array/routines.py

```diff
@@ -1175,13 +1176,14 @@ def _unravel_index_kernel(indices, func_kwargs):
 
 @wraps(np.unravel_index)
 def unravel_index(indices, dims, order='C'):
+    # TODO: deprecate dims as well?
```

Comment:
Do we want to follow NumPy's lead here and rename dims to shape?

Reply:
@shoyer, do you have any thoughts here?

Reply:
This sounds like a good idea to me, though probably not high urgency.